packages/design-tokens/src/js/themes/theme-light-default/colors.ts

```diff
@@ -1,4 +1,24 @@
 /* This file was generated by Supernova, don't change manually */
+export const accent01BackgroundBasic = '#ffbe00';
+export const accent01BackgroundSubtle = '#fff8e5';
+export const accent01BorderBasic = '#a67c00';
+export const accent01BorderSubtle = '#fff8e5';
+export const accent01ContentBasic = '#ffbe00';
+export const accent01ContentSubtle = '#fff8e5';
+export const accent01StateActive = '#cc9800';
+export const accent01StateDefault = '#ffbe00';
+export const accent01StateHover = '#e5ab00';
+
+export const accent02BackgroundBasic = '#0066d8';
+export const accent02BackgroundSubtle = '#e5f0fb';
+export const accent02BorderBasic = '#0066d8';
+export const accent02BorderSubtle = '#e5f0fb';
+export const accent02ContentBasic = '#0052ad';
+export const accent02ContentSubtle = '#e5f0fb';
+export const accent02StateActive = '#3385e0';
+export const accent02StateDefault = '#66a3e8';
+export const accent02StateHover = '#4d94e4';
+
```

Comment:
we will need an object with all accent colors for this to work properly (we don't know the actual number of accent colors in React, so we should just link some object with all of them and use it as Type) 

Will be solved in DS-1880.